org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ClasspathParams.java

```diff
@@ -0,0 +1,123 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.jdt.ls.core.internal;
+
+import org.eclipse.lsp4j.jsonrpc.validation.NonNull;
+import org.eclipse.xtext.xbase.lib.Pure;
+import org.eclipse.xtext.xbase.lib.util.ToStringBuilder;
+
+/*
+ * Representation of a request that client send to server to resolve class path
+ */
+@SuppressWarnings("all")
```

Comment:
i just followed [lsp4j](https://github.com/eclipse/lsp4j/blob/8ac1f87f63857de30f88caf4791de4b847277a28/org.eclipse.lsp4j/src/main/xtend-gen/org/eclipse/lsp4j/CodeLensParams.java), not sure whether it is a good practice as it suppress all the compile warnings